pepdbagent/models.py

```diff
@@ -293,3 +293,25 @@ class SchemaGroupSearchResult(BaseModel):
     limit: int
     offset: int
     results: List[SchemaGroupAnnotation]
+
+
+class TarNamespaceModel(BaseModel):
+    """
+    Geo project model
```

Comment:
is this description and the one below it accurate?

Reply:
no, Good catch!